__mocks__/notifications-node-client.js

```diff
@@ -69,4 +69,17 @@ export class NotifyClient {
       id: "test-sms-return-response-id",
     });
   });
+
+  sendEmail = jest.fn((templateId) => {
```

Comment:
Should we also write a contract test to make sure this behaves as we expect it to when we hit the actual library ?

Reply:
Yes - I'll add that in to the next PR. (Contract tests)